sellotape/sellotape_dj/sellotape_dj/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for sellotape_dj project.
+
+Generated by 'django-admin startproject' using Django 3.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'gtk7i_g+v%afq6dvplwms_-*r3j7uj-k6g4b&&iu#ys-e(ui)0'
```

Comment:
Oh no! you just leaked an important information. Now any one knows your secret by searching on Github "SECRET".
Better set a new SECRET_KEY, remove this one and figure out a way to share it among you guys (maybe in a private chat on Slack and later in `Vault`)
Unless @lioramilbaum / @ifireball think it's OK.

Reply:
Hold your horses, we're still on development environment here, so its ok... in production we would need to ensure the value comes from an environment variable that comes from a vault file or a k8s secret but we're not there yet.

@alonweissfeld you may want to create a future task for this though.